src/utils/registration-helpers.ts

```diff
@@ -769,6 +769,9 @@ export const getOutputName = (item: OutputItem): string => {
 export const userHasAccessRight = (registration: Registration | undefined, operation: RegistrationOperation) =>
   registration?.allowedOperations?.includes(operation) ?? false;
 
+export const userHasAccessRightPartialUpdateButNotUpdate = (registration: Registration) =>
```

Comment:
Litt langt navn kanskje? 😬 
```suggestion
export const userCanOnlyDoPartialUpdate = (registration: Registration) =>
```